org.eclipse.lemminx/src/main/java/org/eclipse/lemminx/services/format/XMLFormatterDocumentNew.java

```diff
@@ -580,7 +580,14 @@ void createTextEditIfNeeded(int from, int to, String expectedContent, List<TextE
 		}
 	}
 
-	public String getIndentSpaces(int level, boolean addLineSeparator) {
+	public boolean shouldCollapseEmptyElement(DOMElement element, SharedSettings sharedSettings) {
+		for (IFormatterParticipant participant : formatterParticipants) {
+			return participant.shouldCollapseEmptyElement(element, sharedSettings);
+		}
+		return false;

```

Comment:
This method loop for the first participant. I think the method should look like this:

```java
public boolean shouldCollapseEmptyElement(DOMElement element, SharedSettings sharedSettings) {
  for (IFormatterParticipant participant : formatterParticipants) {
    if (!participant.shouldCollapseEmptyElement(element, sharedSettings)) {
      return false;
    }
  }
  return true;
}
```